### DISCUSSION TITLE
Keyword search volume

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#35] [Q&A]
How to get amazon keyword search volume in Ads API?

**Comment by elizafarley** [comment]
Hi @Nilanth, unfortunately keyword search volume is not currently available through the Ads API. You can use [search term reports](https://advertising.amazon.com/API/docs/en-us/reporting/v3/report-types#search-term-reports) to understand the performance of keywords associated with your campaign.
